### PR TITLE
Add `yq`

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

- Relates to: [CASMINST-5109](https://jira-pro.it.hpe.com:8443/browse/CASMINST-5109)

#### Issue Type

- RFE Pull Request

Add `yq` for builders to use, requires PackageHub.

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,6 +39,10 @@ RUN zypper --non-interactive install --no-recommends --force-resolution suseconn
     && zypper clean -a
 
 RUN --mount=type=secret,id=SLES_REGISTRATION_CODE suseconnect -r "$(cat /run/secrets/SLES_REGISTRATION_CODE)"
+
+RUN if [ "$TARGETARCH" = 'arm64' ]; then SUSEConnect -p "PackageHub/${SLE_VERSION}/aarch64" ; fi
+RUN if [ $TARGETARCH = 'amd64' ]; then SUSEConnect -p "PackageHub/${SLE_VERSION}/x86_64" ; fi
```
Comment (jpdavis-prof) [inline Dockerfile:44]: Quotes on this var too?

File: Dockerfile [hunk 2]
```diff
@@ -78,6 +83,7 @@ RUN zypper refresh \
         wget \
         which \
         xz-devel \
+        yq \
         zlib-devel \
         && zypper clean -a \
         && suseconnect --cleanup \
```
Comment (jpdavis-prof) [inline Dockerfile:89]: We can leave off this trailing slash.

**Review by jpdavis-prof — APPROVED** [review]
Looks good. Left a couple inline comments.

File: Dockerfile (hunk 2 above)
Comment (rustydb) [reply Dockerfile:89]: Yeah that should go.
I'm surprised it's even here.

**Pull request merged**